guides/common/modules/proc_importing-an-ansible-playbook-by-name.adoc

```diff
@@ -0,0 +1,21 @@
+
+[id="importing-an-ansible-playbook-by-name_{context}"]
+= Importing an Ansible Playbook by Name
+
+You can import an Ansible playbook from collections installed on the {SmartProxy}  by name.
```

Comment:
```suggestion
You can import an Ansible playbook to {Project} from collections installed on {SmartProxy} by name.
You can then use the playbook to invoke jobs using the _Ansible - Run playbook_ job template.
```

Reply:
Do you have a reason not to include the second line in the suggestion?

Reply:
> Do you have a reason not to include the second line in the suggestion?
@Lennonka I think it's common knowledge in this case. The user importing Ansible playbooks would likely know how to use them